sdk/webhooks.md

```diff
@@ -0,0 +1,193 @@
+```
+
+|Parameter	|Type	|Description|
+|---|---|---|
+|`consumerOrgId`	|string	|Consumer Org Id from the console integration URL.|
+|`integrationId`	|string	|Integration Id from the console integration URL.|
+|`body`	|object	|JSON data contains details of the registration. See the sample JSON body for registering a [Journal URL](#sample-json-body-to-register-a-journal-url) or a [Webhook URL](#sample-json-body-to-register-a-webhook-url) for details.|
+
+### Sample JSON Body to Register a Journal URL
+
+The following is a sample JSON request body to register a journal URL. The request includes a name, description, client ID, and delivery type ("JOURNAL" or "WEBHOOK"), as well as an array listing each "event of interest" as an individual object containing the event code and provider ID.
+
+```json
+{
+	"name": "<name>",
+	"description": "<desc>",
+	"client_id": "<client_id>",
+	"delivery_type": "JOURNAL",
+	"events_of_interest": 
+  [
+    {
+		  "event_code": "<event_code>",
+		  "provider_id": "<provider_id>"
+	  }
+  ]
+}
+```
+
+#### Sample JSON Body to Register a Webhook URL
+
+The following is a sample JSON request body to register a webhook URL. You can provide additional events in the `events_of_interest` array by creating an object for each event containing the `event_code` and `provider_id` for the events you are interested in.
+
+```json
+{
+    "name": "<name>",
+    "description": "<desc>",
+    "client_id": "<client_id>",
+    "webhook_url": "<url>",
+    "events_of_interest": [
+    {
+        "event_code": "<event_code>",
+        "provider_id": "<provider_id>"
+    },
+    ...
+  ]
+}
+```
+
+## Get Webhook Registration Details
+
+You can get the registration details for an registration by providing a specific registration ID.
+
+#### Method
+
+```shell
+getWebhookRegistration(consumerOrgId, integrationId, registrationId) ⇒ Promise.<object>
+```
+
+|Parameter	|Type	|Description|
+|---|---|---|
+|`consumerOrgId`	|string	|Consumer Org Id from the console integration URL.|
+|`integrationId`	|string	|Integration Id from the console integration URL.|
+|`registrationId`	|string	|Registration id whose details are to be fetched.|
+
+#### Sample Response
+
+```json
+{
+        "id": 248713,
+        "name": "<name>",
+        "description": "<desc>",
+        "client_id": "<client_id>",
+        "parent_client_id": "<client_id>",
+        "webhook_url": "<url>",
+        "status": "VERIFIED",
+        "type": "APP",
+        "integration_status": "ENABLED",
+        "events_of_interest":
+        [  
+            {
+                "event_code": "<event_code>",
+                "event_label": "<label>",
+                "event_description": "<event_desc>",
+                "provider_id": "<provider_id>",
+                "provider": "<provider_name>",
+                "provider_label": "<provider label>",
+                "event_delivery_format": "cloud_events"
```

Comment:
delivery format can also be adobe_io and more details on thee two types can be provided

Reply:
@sangeetha5491 updated to show both types, where can I find details about each type and when to use them?